autobuild/mac/artifacts/autobuild_mac_3_copy_files.sh

```diff
@@ -23,7 +27,7 @@ echo ""
 
 echo ""
 echo ""
-artifact_deploy_filename=jamulus_${jamulus_buildversionstring}_mac.dmg
+artifact_deploy_filename=jamulus_${jamulus_buildversionstring}_mac${BUILD_SUFFIX}.dmg
```

Comment:
This also needs to be specified on the website (new variable and button on the macOS page).